### DISCUSSION TITLE
Artist is Correct, but has additional artist data not found in Navidrome/Files/etc

### DISCUSSION THREAD

**Discussion by Normanras** [music-assistant/support#2844] [Q&A] [answered]
I couldn't find if this exact thing was asked before, but when using Music Assistant (I'm just using the server via Docker, not the HASS Addon) I see a bunch of other seemingly random artists next to the main artist. 

The good news is that when looking for the artist (in this example, Mogwai) it does find the albums associated with it. If I search for the other artist (Moderat) I _do_ see this album, but this doesn't happen in Navidrome.

![Screenshot 2024-08-30 at 3 34 49 PM](https://github.com/user-attachments/assets/2133e5bf-f688-4680-85a5-3710c51bd72a)
![Screenshot 2024-08-30 at 3 35 04 PM](https://github.com/user-attachments/assets/956a9ba5-f4e9-4003-aad7-19b0b58437ac)

I'm processing everything using beets with MusicBrainz and everything in Navidrome is tagged correctly without the additional artist names (see screenshot). I've recently updated MA so everything is up to date. 

Is there anything I'm missing with my setup that I can re-configure to make sure just the single/correct artist appears? I didn't see anything in issues or other discussions, so forgive me if this was brought up before.

![mogwai-atomic-musicassistant](https://github.com/user-attachments/assets/94265189-ea7d-4979-b9a3-213cbaa6848b)
![mogwai-atomic-navidrome](https://github.com/user-attachments/assets/168ac0cf-2c44-43bd-b41f-8fe25e624a3c)

**Comment by OzGav** [comment, marked as answer]
I assume you are using stable? If so there is a bug that as been fixed in the beta and will arrive in stable soon

**Reply by Normanras** [reply]
I am, and thank you! I looked through issues but I guess I didn't look closely enough. Appreciate the fast response!

**Comment by aquatix** [comment]
Was this fixed in server 2.2.3? If so, I'm still seeing the non-related artists in MA inside HA (I updated to the latest version 2.2.3 and synchronised the library). Thanks in advance!

**Reply by OzGav** [reply]
Do a database reset then to start that from fresh. MA SETTINGS>>CORE>>MUSIC CONTROLLER>> ADVANCED

**Reply by aquatix** [reply]
Cheers, that helped indeed! Artists look good again. For some reason I couldn't find that reset setting, so thanks for the pointer.

**Reply by Normanras** [reply]
Cheers x2! That worked perfectly. Love seeing everything marked correctly again. Thanks :)
